app/Http/Controllers/JobController.php

```diff
@@ -67,12 +67,29 @@ public function managerIndex()
             ->withCount('submitted_applications')
             ->get();
 
+
```

Comment:
Is there anywhere else we'll want to use this logic? (If there is, we can attach this logic to the JobPoster model instead of the controller. If not, this is fine.)

Reply:
Possibly. Currently, not sure where else we can use it.